src/watch-input.internal.ts

```
@@ -0,0 +1,109 @@
+ * @returns Cleanup function
+ */
+export function createCleanupHandler(
+	intervalId: { current: NodeJS.Timeout | null },
+	sessionStartTime: Date,
+	sessionStartTokens: number,
+	sessionStartCost: number,
+	activeBlockForSummary: { current: SessionBlock | null },
+): () => void {
+	return (): void => {
+		if (intervalId.current != null) {
+			clearTimeout(intervalId.current);
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)